crispy_forms/templates/bootstrap4/field.html

```diff
@@ -37,6 +37,11 @@
                 {% include 'bootstrap4/layout/help_text_and_errors.html' %}
             {% elif field|is_file and use_custom_control  %}
                 {% include 'bootstrap4/layout/field_file.html' %}
+            {% elif field|is_selectsingle and use_custom_control %}
+                <div class="{{ field_class }}">
+                     {% crispy_field field 'class' 'custom-control custom-select' %}
```

Comment:
I think we need `custom-select` works just fine on it's own?

Reply:
I'm changing it, you are right: https://getbootstrap.com/docs/4.5/components/forms/#select-menu

The majority of previous controls like the `custom-switch` also have `custom-control`, this might have confused me :-)